src/QMCWaveFunctions/BsplineFactory/SplineC2C.cpp

```diff
@@ -86,7 +86,7 @@ void SplineC2C<ST>::evaluateValue(const ParticleSet& P, const int iat, ValueVect
 #pragma omp parallel
   {
     int first, last;
-    FairDivideAligned(myV.size(), getAlignment<ST>(), omp_get_num_threads(), omp_get_thread_num(), first, last);
+    FairDivideAligned(2 * psi.size(), getAlignment<ST>(), omp_get_num_threads(), omp_get_thread_num(), first, last);
```

Comment:
I'm not completely sure this magic number is from the same cause as all the other bare 2's in this file, but who knows. The line of documentation in the header actually manages to use the word "double" in an ambiguous way until you read the .c

Reply:
Can we replace these 2s with a const named something about complex? I also noted the bare 2s when reading the file.